eng/testing/performance/performance-setup.ps1

```diff
@@ -122,25 +122,45 @@ if ($RunFromPerformanceRepo) {
     $SetupArguments = "--perf-hash $CommitSha $CommonSetupArguments"
     
     robocopy $SourceDirectory $PerformanceDirectory /E /XD $PayloadDirectory $SourceDirectory\artifacts $SourceDirectory\.git
+	if (!$?) {
```

Comment:
I think I would prefer if we used `$LASTEXITCODE` and wrote out its value instead of simply relying on this built-in's true/false value.